code/modules/mob/living/simple_animal/abnormality/zayin/bald.dm

```diff
@@ -74,3 +76,22 @@
 		else
 			icon_state = "bald1"
 
+/mob/living/simple_animal/hostile/abnormality/bald/harvestChem(obj/item/reagent_containers/C, mob/user)
+	harvestPhrase = "<span class='notice'>You squeeze [src]. Some juice drips into [C].</span>"
+	harvestPhraseThirdPerson = "[user] squeezes [src]. Some juice drips into [C]."
+	return ..()
+
+/datum/reagent/abnormality/bald
+	name = "Essence of Baldness"
+	description = "Some weird-looking juice..."
+	color = "#ffffff"
+	specialProperties = list("substance may alter subject physiology")
+	sanityRestore = 1
+
+/datum/reagent/abnormality/bald/on_mob_metabolize(mob/living/L)
+	if(ishuman(L))
+		var/mob/living/carbon/human/balder = L
+		if(balder.hairstyle != "Bald")
+			balder.hairstyle = "Bald"
+			balder.update_hair()
```

Comment:
But why..? Just work on it for the same result. Unless you are planning to grief with it by putting reagent into food...